utils/hct/pkgesbuild/status-email-template.html

```diff
@@ -0,0 +1,20 @@
+<style>
```

Comment:
Is html for emails different or is it just my obsessive-compulsiveness that makes me long for <html><head><body> tags and new <p>'s instead of double <br/>'s?

Reply:
This is just for the email body - it will be embedded in <body>tag etc. by the Azure Pipelines Send Email task. For the double <br/> I have no excuse ;).